client/src/Authentication/AuthScreen.tsx

```diff
@@ -46,10 +46,12 @@ export default function AuthScreen({
     }
 
     const signupHandler = async () => {
+        // Before we migrate the backend to a server, use the internal IP address of your machine
+        const internalIP: string = "10.136.195.96";
```

Comment:
Is there a way to make this read from the .env file as well instead of manually changing it in the .env file and the frontend code? Maybe we could move the .env file to the project root and have both client and api read from `../.env` instead of `./.env`